ui/app/store/actions.js

```diff
@@ -817,16 +817,6 @@ export function estimateGasMethod ({
   })
 }
 
-export async function estimateGasFromTxParams (txParams) {
-  const backgroundState = await updateMetamaskStateFromBackground()
-  const blockGasLimit = backgroundState.currentBlockGasLimit
-  return estimateGasMethod({
```

Comment:
Note that the above `estimateGasMethod` is now also unused, as this was the sole place it was invoked.

There is another _parameter_ in this file also called `estimateGasMethod`, but that is distinct from this function.

Reply:
Removed in: 82a75fc

Reply:
Err, I mean 2ffcffb